src/components/ArticleComponents/Codebox/index.tsx

```diff
@@ -20,49 +20,53 @@ const replaceLanguages = (language: string) =>
     .replace(/mjs|cjs|javascript/i, 'js')
     .replace(/console|shell/i, 'bash');
 
-const replaceLabelLanguages = (language: string) =>
-  language
-    .replace(/javascript/i, 'cjs')
-    .replace(/console|shell/i, 'bash')
-    .toUpperCase();
-
 const Codebox = ({ children: { props } }: Props): JSX.Element => {
   const [parsedCode, setParsedCode] = useState('');
   const [copied, copyText] = useCopyToClipboard();
-
+  const [langIndex, setLangIndex] = useState(0);
   // eslint-disable-next-line react/prop-types
   const className = props.className || 'text';
-
   // Language Matches in class
   const matches = className.match(/language-(?<lang>.*)/);
+  const languageOptions = (matches?.groups?.lang || 'text').split('|');
+  const language = languageOptions[langIndex];
+  const codeArray = props.children
+    ? props.children.toString().split('-------\n')
+    : [''];
 
-  // Language name from classname
-  const language = matches?.groups?.lang || 'text';
-
-  // Actual Code into a stringified format
-  const stringCode = props.children?.toString() || '';
-
-  const handleCopyCode = async (event: React.MouseEvent<HTMLButtonElement>) => {
+  const handleCopyCode = async (
+    event: React.MouseEvent<HTMLButtonElement>
+  ): Promise<void> => {
```

Comment:
I think we don't need to add an explicit return type here :)

Reply:
Yeah 😐